javascript/javascript1/week3/series-duration-of-my-life/series-duration-of-my-life.js

```diff
@@ -0,0 +1,47 @@
+const seriesDurations = [
+  {
+    title: "Game of thrones",
+    days: 3,
+    hours: 1,
+    minutes: 0,
+  },
+  {
+    title: "Sopranos",
+    days: 3,
+    hours: 14,
+    minutes: 0,
+  },
+  {
+    title: "The Wire",
+    days: 2,
+    hours: 12,
+    minutes: 0,
+  },
+  {
+    title: "Breaking Bad",
+    days: 16,
+    hours: 21,
+    minutes: 0,
+  },
+];
+
+const perOfSeriesInYear = (obj) => {
```

Comment:
This function is a bit long and gives back 2 different things - information per series duration and total. It could be split into 2 separate functions, or you could add some comments explaining the different parts of the code. No need to do anything now, just keep in mind for the future, those 2 approaches.